examples/local/lib/utils.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Wait for the given number of tablets to show up
+# for the keyspace/shard in the topology server.
+# Example (wait for 2 tables in commerce/0):
+#   wait_for_shard_tablets commerce 0 2
+function wait_for_shard_tablets() {
+	if [[ -z ${1} || -z ${2} || -z ${3} ]]; then
+		fail "A keyspace, shard, and number of tablets must be specified when waiting for tablets to come up"
+	fi
+	keyspace=${1}
+	shard=${2}
+	num_tablets=${3}
+
+	for _ in $(seq 0 200); do
```

Comment:
Since we also have a tablet id 200, this has always been a bit confusing to me! Since we are refactoring anyway, can we please use a constant here and below to be clear what this range upper value is for?